src/app/app.component.ts

```diff
@@ -10,26 +10,29 @@ export class AppComponent implements OnInit {
   public adjustFooter = false;
   public isMobile: boolean;
   public isHome: boolean;
+  public isPageNotFound: boolean;
 
   constructor(private router: Router) {
     router.events.subscribe((val) => {
       if (val instanceof NavigationEnd) {
         this.isHome = val.url === "/" ? true : false;
+        this.isPageNotFound = val.urlAfterRedirects === "/404" ? true : false;
       }
     });
 
-    this.bindEvents();
+    // this.bindEvents();
   }
 
   ngOnInit() {
     this.isMobile = window.innerWidth < 991 ? true : false;
-    this.isHome = window.location.pathname === "/" ? true : false;
+    this.isHome = window.location.pathname === "/new" ? true : false;
+    this.isPageNotFound = window.location.pathname === "/404" ? true : false;
   }
 
-  bindEvents() {
-    window.addEventListener("load", () => {
-      // tslint:disable-next-line: no-unused-expression
-      this.router.url !== "/" && this.router.navigate(["/"]);
-    });
-  }
+  // bindEvents() {
+  //   window.addEventListener("load", () => {
+  //     // tslint:disable-next-line: no-unused-expression
+  //     this.router.url !== "/" && this.router.navigate(["/"]);
+  //   });
+  // }
```

Comment:
Double checking that this can indeed be commented out and it's not accidentally left here from debugging!

Reply:
Yes, I agree we can remove it. We can always restore it from the history if something is wrong.